hub/core/dataset/view_entry.py

```diff
@@ -18,7 +18,7 @@ def get(self, key: str, default: Optional[Any] = None):
     @property
     def id(self) -> str:
         """Returns id of the view."""
-        return self.info["id"]
+        return self.info["id"].split("]")[-1]
```

Comment:
why splitting by "]" ends with view id? what is the form?

Reply:
Internally, when a view is stored in the users folder, we prefix it with [org_id][ds_name] to avoid conflict with views from other datasets with same id.

Reply:
got it!